sanity-io/structured-content-2022
# "Venue names" component for /home

# "Venue names" component for /home

## Intent

Create structure-only component for /home based off Figma sketches:
![image](https://user-images.githubusercontent.com/6001131/154677355-d7c4e9f6-5be2-41ed-ac0e-fecbc69b9c65.png)


## Description

This PR just gets the Venues and displays the names as specified by the Figma sketches. As per our arrangement, no styling has been added. 

## Testing this PR

1. `docker-compose up`
2. Verify that all the venues mentioned in the Figma sketch is listed on [/home](http://localhost:3000/home)
3. Add the CSS (and modify the markup if necessary) so that this component matches that of the Figma sketches
